dev-haskell/bson/bson-0.4.0.1-r2.ebuild

```
@@ -29,8 +29,3 @@ DEPEND="${RDEPEND}
 		>=dev-haskell/test-framework-0.4
 		>=dev-haskell/test-framework-quickcheck2-0.2 )
 "
-
-src_configure() {
-	haskell-cabal_src_configure \
-		--flag=_old-network
-}
```

Don't remove this completely, instead switch it from always-on to always-off (add `-` between `--flag` and the flag name), which matches the `#hackport` line.

Any flags that are listed in the package's `.cabal` file should be explicitly listed in `haskell-cabal_src_configure` (either always-on, always-off, or toggled via e.g. `USE` flags).

```suggestion
src_configure() {
	haskell-cabal_src_configure \
		--flag=-_old-network
}
```

---

@hololeap done

---

@hololeap done